src/Repository/EventRepository.php

```diff
@@ -515,6 +515,28 @@ public function queryCountByMonth(Adherent $referent, int $months = 5): QueryBui
         ;
     }
 
+    public function countParticipantsInReferentManagedArea(Adherent $referent): int
+    {
+        $this->checkReferent($referent);
+
+        $results = $this->createQueryBuilder('event')
+            ->select('event.participantsCount')
+            ->innerJoin('event.referentTags', 'tag')
+            ->where('tag IN (:tags)')
+            ->andWhere('event.committee IS NOT NULL')
+            ->andWhere("event.status = '".Event::STATUS_SCHEDULED."'")
+            ->andWhere('event.organizer != :referent')
```

Comment:
Technically a referent could be a committee supervisor or host and also committee event organiser. And in this case, it's not correct to add this condition.
To be sure that we get only committee  events it's enough to check that committee is not null as you did here https://github.com/EnMarche/en-marche.fr/pull/2474/files#diff-265bef353d2bcbcdada5c1736d191b53R526